manifests/vrrp/sync_group.pp

```diff
@@ -25,6 +27,7 @@
 define keepalived::vrrp::sync_group (
   $group,
   Optional[Stdlib::Absolutepath] $notify_script_master_rx_lower_pri = undef,
+  Optional[Array] $track_script                                     = undef,
```

Comment:
instead of undef, can you default to an empty array and change the datatype from Optional to Array? that would follow best practice guidelines. can you enforce the datatype in the array?

Reply:
Fixed as requested.